plugins/ufm_events_grafana_dashboard_plugin/build/Dockerfile

```diff
@@ -7,15 +7,19 @@ ARG BASE_PATH=/opt/ufm/ufm_plugin_${PLUGIN_NAME}
 ARG SRC_BASE_DIR=${PLUGIN_NAME}_plugin
 ARG ETC_ALTERNATIVE_PATH=/var/etc
 ARG SUPERVISOR_PATH=${ETC_ALTERNATIVE_PATH}/supervisor
+ARG LOKI_VERSION=3.1.0
```

Comment:
Please also update the readme with some explanation of this changes

Reply:
I've a separate task to update the README & HLD to reflect all these and future changes